tests/framework/data/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+
+
+# Uncomment to rebuild the targets
+if(UNIX)
+#     add_library(dummy_library_elf_64 SHARED stub.cpp)
```

Comment:
We should really add a CMake define so we can trigger this without rebuilding.

Reply:
The issue I found is that this doesn't work on all compilers as multi-arch builds really aren't well supported by cmake (or in general). I was tempted to not have any 'rebuilding' code but thought that they should at least 'exist' in the source tree in case for whatever reason they "do" need to be rebuilt. 

This applies for windows doubly, since it doesn't have a magic flag you can just set and have it work.